### PR TITLE
Removes turfs global list to speed up server startup

### PR DESCRIPTION
**Pull request by caelaislinn**
See commits

### CODE REVIEW

**Comment by bloxgate** [comment]
This changes global variables so you're going to have to run the script and get the new checksum for them.

**Comment by caelaislinn** [comment]
I was waiting for Travis to do it for me

File: test/run-test.sh [hunk 1]
```diff
@@ -205,7 +205,7 @@ function run_code_tests {
     run_test_fail "no invalid spans" "grep -En \"<\s*span\s+class\s*=\s*('[^'>]+|[^'>]+')\s*>\" **/*.dm"
     run_test "code quality checks" "test/check-paths.sh"
     run_test "indentation check" "awk -f tools/indentation.awk **/*.dm"
-    run_test "check changelog example unchanged" "md5sum -c - <<< '79e058ac02ed52aad99a489ab4c8f75b *html/changelogs/example.yml'"
+    run_test "check changelog example unchanged" "md5sum -c - <<< '38c768b1cfcce663fae52e86e59883ac *html/changelogs/example.yml'"
```
Comment (bloxgate) [inline test/run-test.sh]: You've changed the wrong sum here. That's the example changelog sum. That should never change.

**Review by bloxgate — DISMISSED** [review]

File: test/run-test.sh (hunk 1 above)
Comment (caelaislinn) [reply test/run-test.sh]: Oops hehe

**Review by BDpuffy420 — APPROVED** [review]
Fixes have been made

**Pull request merged**